models/convert/cadence.go

```diff
@@ -36,7 +36,7 @@ func ParseCadenceArgument(param string) (cadence.Value, error) {
 	typ := parts[1]
 	val := parts[2]
 
-	// Now, we can switch on the type and parse accordingly.
+	// Switch on the type and parse accordingly.
```

Comment:
To make the point again: this reads less nice. The previous version is like a flow/story that takes the reader through the sequence of events and connects them together. The new one is just saying "the next statement does this", which is just redundant with the code.